---
title: Lint Rule js/defaultExportSameBasename
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/defaultExportSameBasename,
	parent: lint-rules,
	title: js/defaultExportSameBasename
}
---

# js/defaultExportSameBasename

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
export default function test() {}
```
```typescript
export default class Test {}
```
## Valid
```typescript
export default function foo() {}
```
```typescript
export default class Foo {}
```
```typescript
export default 'rome';
```
