---
title: Lint Rule js/doubleEquals
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/doubleEquals,
	parent: lint-rules,
	title: js/doubleEquals
}
---

# js/doubleEquals

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
foo == bar
```
## Valid
```typescript
foo == null
```
```typescript
foo != null
```
```typescript
null == foo
```
```typescript
null != foo
```
