---
title: Lint Rule js/duplicateImportSource
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/duplicateImportSource,
	parent: lint-rules,
	title: js/duplicateImportSource
}
---

# js/duplicateImportSource

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
import	foo	from	'./testdummy.ts';
import	{bar}	from	'./testdummy.ts';
import	type	{fooType}	from	'./testdummy.ts';

const	typedFoo:	fooType	=	{
	type:	'foo'
};
```
## Valid
