---
title: Lint Rule js/emptyMatches
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/emptyMatches,
	parent: lint-rules,
	title: js/emptyMatches
}
---

# js/emptyMatches

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
let a = /a*/
```
```typescript
let b = /a*(abc)?[1,2,3]*/
```
## Valid
```typescript
let a = /a*(abc)+[1,2,3]?/
```
```typescript
let b = /a+(abc)*/
```
