---
title: Lint Rule js/emptyBlocks
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/emptyBlocks,
	parent: lint-rules,
	title: js/emptyBlocks
}
---

# js/emptyBlocks

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
if (foo) {}
```
```typescript
if (foo) {
	// foo;
} else {}
```
## Valid
```typescript
if (foo) foo;
```
```typescript
if (foo) {
	foo;
}
```
```typescript
if (foo) {
	// empty
}
```
