---
title: Lint Rule js/getterReturn
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/getterReturn,
	parent: lint-rules,
	title: js/getterReturn
}
---

# js/getterReturn

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
class p {
	get name() {
		console.log('hello');
	}
}
```
```typescript
let p = {
	get name() {
		console.log('hello');
	},
};
```
## Valid
```typescript
let p = {};
Object.defineProperty(p, {
	get: function () {
		console.log('hello');
	},
});
```
