---
title: Lint Rule js/importDefaultBasename
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/importDefaultBasename,
	parent: lint-rules,
	title: js/importDefaultBasename
}
---

# js/importDefaultBasename

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
import foo from './bar';
```
## Valid
```typescript
import foo from './foo';
```
