---
title: Lint Rule js/negationElse
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/negationElse,
	parent: lint-rules,
	title: js/negationElse
}
---

# js/negationElse

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
if (!true) {consequent;} else {alternate;}
```
```typescript
!true ? consequent : alternate
```
## Valid
```typescript
if (!true) {consequent;}
```
```typescript
true ? consequent : alternate
```
