---
title: Lint Rule js/noArguments
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noArguments,
	parent: lint-rules,
	title: js/noArguments
}
---

# js/noArguments

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
function f() {
	console.log(arguments);
}
```
```typescript
(function () {
	console.log(arguments);
})();
```
```typescript
class C {
	fn() {
		console.log(arguments);
	}
}
```
```typescript
const o = {
	fn() {
		console.log(arguments);
	},
};
```
## Valid
