---
title: Lint Rule js/noAsyncPromiseExecutor
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noAsyncPromiseExecutor,
	parent: lint-rules,
	title: js/noAsyncPromiseExecutor
}
---

# js/noAsyncPromiseExecutor

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
new Promise(async function foo() {})
```
```typescript
new Promise(async () => {})
```
```typescript
new Promise(((((async () => {})))))
```
## Valid
```typescript
new Promise(() => {})
```
```typescript
new Promise(() => {}, async function unrelated() {})
```
```typescript
class Foo {} new Foo(async () => {})
```
