---
title: Lint Rule js/noCatchAssign
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noCatchAssign,
	parent: lint-rules,
	title: js/noCatchAssign
}
---

# js/noCatchAssign

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
try { } catch (e) { e; e = 10; }
```
```typescript
try { } catch (ex) { console.log('test'); ex = 10; }
```
```typescript
try { } catch (ex) { [ex, test] = []; }
```
```typescript
try { } catch ({message, name}) { message = 'test'; name = 10; }
```
```typescript
try { } catch (ex) { ({x: ex = 0} = {}); }
```
```typescript
try { } catch (ex) { let a; ({x: a = ex = 0} = {}); }
```
## Valid
```typescript
try { } catch (e) { three = 2 + 1; }
```
```typescript
try { } catch ({e}) { this.something = 2; }
```
```typescript
function foo() { try { } catch (e) { return false; } }
```
