---
title: Lint Rule js/noCompareNegZero
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noCompareNegZero,
	parent: lint-rules,
	title: js/noCompareNegZero
}
---

# js/noCompareNegZero

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
(1 >= -0)
```
## Valid
```typescript
(1 >= 0)
```
