---
title: Lint Rule js/noCommaOperator
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noCommaOperator,
	parent: lint-rules,
	title: js/noCommaOperator
}
---

# js/noCommaOperator

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
(0, 1, 2)
```
```typescript
test(), rome()
```
## Valid
```typescript
foo(0, 1, 2)
```
```typescript
[1, 2,]
```
```typescript
[1,,,3]
```
```typescript
let a, b, c;
```
