---
title: Lint Rule js/noCondAssign
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noCondAssign,
	parent: lint-rules,
	title: js/noCondAssign
}
---

# js/noCondAssign

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
for (let i = 1; i = 10; i++) {
	console.log('foo');
}
```
```typescript
if (foo = 'bar') {
	console.log('foo');
}
```
```typescript
while (foo = 'bar') {
	console.log('foo');
}
```
```typescript
do {
	console.log('foo');
} while (foo = 'bar')
```
```typescript
(foo = bar) ? foo() : bar();
```
## Valid
```typescript
while ((foo = foo.bar) !== undefined) {
	console.log(foo);
}
```
```typescript
if (foo++ === 3) {
	console.log(foo);
}
```
```typescript
foo = bar ? foo() : bar();
```
