---
title: Lint Rule js/noDuplicateGroupNamesInRegularExpressions
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noDuplicateGroupNamesInRegularExpressions,
	parent: lint-rules,
	title: js/noDuplicateGroupNamesInRegularExpressions
}
---

# js/noDuplicateGroupNamesInRegularExpressions

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
/(?<month>[0-9])-(?<year>[0-9])-(?<month>[0-9])-(?<year>[0-9])-(?<day>[0-9])-([0-9])-(?<month>[0-9])/
```
## Valid
