---
title: Lint Rule js/noDuplicateKeys
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noDuplicateKeys,
	parent: lint-rules,
	title: js/noDuplicateKeys
}
---

# js/noDuplicateKeys

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
const foo = {
	test: true,
	test2: true,
	test: false,
};
```
## Valid
