---
title: Lint Rule js/noDupeArgs
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noDupeArgs,
	parent: lint-rules,
	title: js/noDupeArgs
}
---

# js/noDupeArgs

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
function hello(a, a) {
	//
}
```
```typescript
const hello = (a, a) => {
	//
}
```
```typescript
const hello = function (a, a) {
	//
}
```
## Valid
```typescript
function foo(foo) {
	console.log(foo)
}
```
```typescript
const foo = 'test'
function bar(foo) {
	console.log(foo)
}
```
