---
title: Lint Rule js/noEmptyCharacterClass
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noEmptyCharacterClass,
	parent: lint-rules,
	title: js/noEmptyCharacterClass
}
---

# js/noEmptyCharacterClass

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
let foo = /^abc[]/;foo;
```
```typescript
let foo = /foo[]bar/;foo;
```
```typescript
;if (foo.match(/^abc[]/)) { foo; }
```
```typescript
let foo = /[]]/;foo;
```
```typescript
let foo = /\[[]/;foo;
```
```typescript
let foo = /\[\[\]a-z[]/;foo;
```
## Valid
```typescript
let foo = /^abc[a-zA-Z]/;foo;
```
```typescript
let regExp = new RegExp('^abc[]');regExp;
```
```typescript
let foo = /^abc/;foo;
```
```typescript
let foo = /[\[]/;foo;
```
```typescript
let foo = /[\]]/;foo;
```
```typescript
let foo = /[a-zA-Z\[]/;foo;
```
```typescript
let foo = /[[]/;foo;
```
```typescript
let foo = /[\[a-z[]]/;foo;
```
```typescript
let foo = /[\-\[\]\/\{\}\(\)\*\+\?\.\\^\$\|]/g;foo;
```
```typescript
let foo = /[\]]/uy;foo;
```
```typescript
let foo = /[\]]/s;foo;
```
```typescript
let foo = /\[]/;foo;
```
