---
title: Lint Rule js/noFunctionAssign
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noFunctionAssign,
	parent: lint-rules,
	title: js/noFunctionAssign
}
---

# js/noFunctionAssign

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
function foo() {}; foo = bar;
```
```typescript
function foo() { foo = bar; }
```
```typescript
foo = bar; function foo() { };
```
```typescript
[foo] = bar; function foo() { };
```
```typescript
({x: foo = 0} = bar); function foo() { };
```
```typescript
function foo() { [foo] = bar; }
```
```typescript
(function() { ({x: foo = 0} = bar); function foo() { }; })();
```
## Valid
```typescript
function foo() { var foo = bar; }
```
```typescript
function foo(foo) { foo = bar; }
```
```typescript
function foo() { var foo; foo = bar; }
```
```typescript
var foo = () => {}; foo = bar;
```
```typescript
var foo = function() {}; foo = bar;
```
```typescript
var foo = function() { foo = bar; };
```
```typescript
import bar from 'bar'; function foo() { var foo = bar; }
```
