---
title: Lint Rule js/noImportAssign
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noImportAssign,
	parent: lint-rules,
	title: js/noImportAssign
}
---

# js/noImportAssign

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
import x from 'y';
x=1;
```
```typescript
import x from 'y';
[x]=1;
```
```typescript
import x from 'y';
({x}=1);
```
```typescript
import x from 'y';
x++
```
```typescript
import x from 'y';
[...x]=1;
```
```typescript
import x from 'y';
({...x}=1);
```
```typescript
import x from 'y';
for (x in y);
```
```typescript
import x from 'y';
x+=1
```
```typescript
import * as x from 'y';
x=1;
```
```typescript
import {x} from 'y';
x=1;
```
## Valid
