---
title: Lint Rule js/noMultipleSpacesInRegularExpressionLiterals
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noMultipleSpacesInRegularExpressionLiterals,
	parent: lint-rules,
	title: js/noMultipleSpacesInRegularExpressionLiterals
}
---

# js/noMultipleSpacesInRegularExpressionLiterals

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
/foo  bar/
```
## Valid
```typescript
/foo {2}bar/
```
