---
title: Lint Rule js/noNestedTernary
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noNestedTernary,
	parent: lint-rules,
	title: js/noNestedTernary
}
---

# js/noNestedTernary

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
let thing = foo ? bar : baz === qux ? quxx : foobar;
```
```typescript
let thing = foo ? bar ? boo : foo : baz ? boo : foo;
```
```typescript
foo ? baz === qux ? quxx() : foobar() : bar();
```
## Valid
```typescript
let thing = foo ? bar : foobar;
```
```typescript
let thing = foo ? bar || boo : foo || bar;
```
```typescript
let thing = foo ? bar && boo : foo && bar;
```
```typescript
let thing = foo || baz ? bar || boo : foo || bar;
```
```typescript
let thing = foo && baz ? bar || boo : foo && bar;
```
```typescript
let thing = foo || baz ? bar || boo : foo && bar;
```
```typescript
if (foo) {
	thing = bar;
} else if (baz === qux) {
	thing = quxx;
} else {
	thing = foobar;
}
```
