---
title: Lint Rule js/noPosixInRegularExpression
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noPosixInRegularExpression,
	parent: lint-rules,
	title: js/noPosixInRegularExpression
}
---

# js/noPosixInRegularExpression

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
const pattern = /[[:alpha:]]/
```
```typescript
const pattern = /[[.ch.]]/
```
## Valid
