---
title: Lint Rule js/noSetterReturn
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noSetterReturn,
	parent: lint-rules,
	title: js/noSetterReturn
}
---

# js/noSetterReturn

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
class p {
	set name(value) {
		if (!value) {
			return 'wrong';
		}
	}
}
```
```typescript
class p {
	static set name(value) {
		if (!value) {
			return 'wrong';
		}
	}
}
```
```typescript
let p = {
	set name(value) {
		if (!value) {
			return 'wrong';
		}
	}
};
```
## Valid
```typescript
class p {
	set name(value) {
		if (!value) {
			return;
		}
	}
}
```
