---
title: Lint Rule js/noShorthandArrayType
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noShorthandArrayType,
	parent: lint-rules,
	title: js/noShorthandArrayType
}
---

# js/noShorthandArrayType

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
let valid: Array<foo>;
let invalid: bar[];
```
## Valid
