---
title: Lint Rule js/noShadowRestrictedNames
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noShadowRestrictedNames,
	parent: lint-rules,
	title: js/noShadowRestrictedNames
}
---

# js/noShadowRestrictedNames

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
function NaN() {}
```
```typescript
let Set;
```
```typescript
try {  } catch(Object) {}
```
```typescript
!function Array() {}
```
```typescript
function test(JSON) {console.log(JSON)}
```
## Valid
