---
title: Lint Rule js/noTemplateCurlyInString
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noTemplateCurlyInString,
	parent: lint-rules,
	title: js/noTemplateCurlyInString
}
---

# js/noTemplateCurlyInString

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
const user = 'Faustina';
               const helloUser = 'Hello, ${user};
```
## Valid
