---
title: Lint Rule js/noUnusedTemplateLiteral
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noUnusedTemplateLiteral,
	parent: lint-rules,
	title: js/noUnusedTemplateLiteral
}
---

# js/noUnusedTemplateLiteral

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
const foo = `bar`
```
```typescript
const foo = `bar `
```
## Valid
```typescript
const foo = `bar
`
```
```typescript
const foo = `"bar"`
```
```typescript
const foo = `'bar'`
```
