---
title: Lint Rule js/noUnusedVariables
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noUnusedVariables,
	parent: lint-rules,
	title: js/noUnusedVariables
}
---

# js/noUnusedVariables

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
const a = 4;
```
```typescript
let a = 4;
```
```typescript
function foo(myVar) {
	console.log('foo');
};
```
```typescript
function foo(myVar) {
	console.log('foo');
}
foo();
```
```typescript
const foo = (b) => {
	console.log('foo');
};
```
## Valid
```typescript
function foo(_, b) {
	console.log(b)
};
foo();
```
```typescript
const foo = (_, b) => {
	console.log(b)
};
foo();
```
