---
title: Lint Rule js/noUnsafeFinally
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noUnsafeFinally,
	parent: lint-rules,
	title: js/noUnsafeFinally
}
---

# js/noUnsafeFinally

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
function greet1() {
	try {
		throw new Error('Try')
	} catch(err) {
		throw err;
	} finally {
		return 1;
	}
}
```
```typescript
function greet2() {
	try {
		throw new Error('Try')
	} catch(err) {
		throw err;
	} finally {
		break;
	}
}
```
```typescript
function greet3() {
	try {
		throw new Error('Try')
	} catch(err) {
		throw err;
	} finally {
		continue;
	}
}
```
```typescript
function greet4() {
	try {
		throw new Error('Try')
	} catch(err) {
		throw err;
	} finally {
		throw new Error('Finally');
	}
}
```
## Valid
