---
title: Lint Rule js/noVar
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/noVar,
	parent: lint-rules,
	title: js/noVar
}
---

# js/noVar

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
var foobar;
foobar
```
## Valid
