---
title: Lint Rule js/preferFunctionDeclarations
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/preferFunctionDeclarations,
	parent: lint-rules,
	title: js/preferFunctionDeclarations
}
---

# js/preferFunctionDeclarations

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
const foo = function () {};
```
```typescript
const foo = () => {};
```
```typescript
const foo = () => {function bar() {this;}};
```
## Valid
```typescript
const foo = () => {this;};
```
```typescript
const foo: string = function () {};
```
