---
title: Lint Rule js/preferWhile
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/preferWhile,
	parent: lint-rules,
	title: js/preferWhile
}
---

# js/preferWhile

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
for (; x.running;) {
  x.step();
}
```
```typescript
for (;;) {
  doSomething();
}
```
## Valid
