---
title: Lint Rule js/preferBlockStatements
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/preferBlockStatements,
	parent: lint-rules,
	title: js/preferBlockStatements
}
---

# js/preferBlockStatements

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
if (x) x;
```
```typescript
if (x) {
	x;
} else y;
```
```typescript
if (x) {
	x
} else if (y) y;
```
```typescript
for (;;);
```
```typescript
for (p in obj);
```
```typescript
for (x of xs);
```
```typescript
do; while (x);
```
```typescript
while (x);
```
```typescript
with (x);
```
## Valid
