---
title: Lint Rule js/restrictedGlobals
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/restrictedGlobals,
	parent: lint-rules,
	title: js/restrictedGlobals
}
---

# js/restrictedGlobals

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
console.log(event);
```
```typescript
foo(event)
```
## Valid
```typescript
function foo(event) {
	console.info(event);
}
```
