---
title: Lint Rule js/singleVarDeclarator
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/singleVarDeclarator,
	parent: lint-rules,
	title: js/singleVarDeclarator
}
---

# js/singleVarDeclarator

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
let foo, bar;
```
## Valid
```typescript
for (let i = 0, x = 1; i < arr.length; i++) {}
```
