---
title: Lint Rule js/sortImportExportSpecifiers
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/sortImportExportSpecifiers,
	parent: lint-rules,
	title: js/sortImportExportSpecifiers
}
---

# js/sortImportExportSpecifiers

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
import {b, a, c, D} from 'mod';
```
```typescript
import {b as A, a as C, B} from 'mod';
```
```typescript
import {c, b as b2, b as b1, b} from 'mod';
```
```typescript
export {b, a, c, D} from 'mod';
```
```typescript
export {b as A, a as C, B} from 'mod';
```
```typescript
export {c, b as b2, b as b1, b} from 'mod';
```
```typescript
export {b, a, c, D};
```
```typescript
export {b as A, a as C, B};
```
```typescript
export {c, b as b2, b as b1, b};
```
## Valid
