---
title: Lint Rule js/sparseArray
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/sparseArray,
	parent: lint-rules,
	title: js/sparseArray
}
---

# js/sparseArray

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
[1,,2]
```
## Valid
