---
title: Lint Rule js/unsafeNegation
layout: layouts/page.njk
description: MISSING DOCUMENTATION
eleventyNavigation: {
	key: lint-rules/js/unsafeNegation,
	parent: lint-rules,
	title: js/unsafeNegation
}
---

# js/unsafeNegation

MISSING DOCUMENTATION

<!-- EVERYTHING BELOW IS AUTOGENERATED. SEE SCRIPTS FOLDER FOR UPDATE SCRIPTS -->


## Examples
## Invalid
```typescript
!1 in [1,2]
```
## Valid
